apps/demos/Demos/Toolbar/Adaptability/Angular/app/app.component.ts

```diff
@@ -36,6 +37,8 @@ if (!/localhost/.test(document.location.host)) {
 })
 
 export class AppComponent {
+  stylingMode = isMaterialBased() ? 'text' : undefined;
```

Comment:
It's not a public API and should not be used in demos. Please search through the written demos, find the way how this check should be implemented and use the same way.

Reply:
[6d02fe8](https://github.com/DevExpress/DevExtreme/pull/27509/commits/6d02fe8dd0db7abe9c3a3bf2a4517a2ef078836f)